### PR TITLE
Pom file updates

### PR DESCRIPTION
**Pull request by nikhilajudiya**
# type(scope): if this commit is applied, it will...
i.e. fix(login): [more detailed message]

# Why was this change made?

Feature request? A bug was found? Doing some refactoring? Let us know why this should be in our codebase.

# Links to any relevant tickets, articles, or other resources

closes https://github.com/SchemaApp/SchemaApp/issues/11869 
closes https://github.com/SchemaApp/SchemaApp/issues/11868 
# Other Notes

Did you fix any additional issues or do you have special notes for the reviewer? Put them here.

# Suggested Test for Reviewer

1. Login to Schema app dashboard
2. Go to 
3. Do action
4. See result

### CODE REVIEW

**Review by joelacummings — CHANGES_REQUESTED** [review]
I'm noticing a problem when there is highlight.js and editor markup for the page. I don't see it pulling in highlight.js markup for the the homepage on our test environment. I think there is an issue around this. If there is markup under __highlighter_js and under the base directory it can either combine them or it can include two separate json-ld blocks. 

There is also a codacy around instantiating string objects.

**Comment by nikhilajudiya** [comment]
Hi @joelacummings I can see JSONL-D added correct for home page - 
image

File: all/pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>com.schemaapp</groupId>
         <artifactId>schemaapp-aem</artifactId>
-        <version>1.3.1</version>
+        <version>1.0.7-SNAPSHOT</version>
```
Comment (joelacummings) [inline all/pom.xml:27]: Can we make this 1.4 and not a snapshot so we can release this more broadly?

File: core/src/main/java/com/schemaapp/core/services/impl/CDNDataAPIServiceImpl.java [hunk 2]
```diff
@@ -115,112 +125,181 @@ private void processPage(ResourceResolver resolver,
...
-                    }
-                    graphJsonDataArray.put(graphJsonDataObject);
-                } else {
-                    graphJsonDataArray.put(graphJsonDataObject);
-                    if (graphJsonData != null)
-                        graphJsonDataArray.put(graphJsonData);
-                }
-                LOG.info(String.format("CDN data response:: javascript :: %s", response));
-                return graphJsonDataArray;
+    private Object processJavaScriptCDNData(
```
Comment (joelacummings) [inline core/src/main/java/com/schemaapp/core/services/impl/CDNDataAPIServiceImpl.java:251]: add a docstring here.

**Review by joelacummings — CHANGES_REQUESTED** [review]
Manual testing was fine, code needs to have docstrings and we need a release version.

**Review by joelacummings — APPROVED** [review]

**Pull request merged**